pkg/reconciler/taskrun/taskrun.go

```diff
@@ -185,7 +184,9 @@ func (c *Reconciler) ReconcileKind(ctx context.Context, tr *v1beta1.TaskRun) pkg
 	// Reconcile this copy of the task run and then write back any status
 	// updates regardless of whether the reconciliation errored out.
 	if err = c.reconcile(ctx, tr, taskSpec, rtr); err != nil {
-		logger.Errorf("Reconcile error: %v", err.Error())
```

Comment:
Looking at this again, I don't totally understand why this changed.  I think we want to print any non-nil error regardless of whether it's a multierror or not?

Reply:
Correct, changed it!